lizmap/modules/admin/locales/en_US/admin.UTF-8.properties

```
@@ -367,8 +367,15 @@ project.modal.button.close=Close
 project.list.no.hidden.column.content=No extra content for this project in the following hidden table columns
 
 
-landingPageContent.top=Content at the top of the landing page
-landingPageContent.bottom=Content at the bottom of the landing page
+landingPageContent.authed=Content for authentified users
+landingPageContent.unauthed=Content for unauthentified users
+landingPageContent.unauthed.position=Add content for unauthentified user to authentified users ?
+landingPageContent.group.top=Top of the landing page
+landingPageContent.group.bottom=Bottom of the landing page
+landingPageContent.unauthed.position.before=before content for authentified
+landingPageContent.unauthed.position.after=after content for authentified
+landingPageContent.unauthed.position.no=no
```

```suggestion
landingPageContent.unauthed.position.before=Before content for authentified
landingPageContent.unauthed.position.after=After content for authentified
landingPageContent.unauthed.position.no=No
```
Same, HIG guidelines, radio buttons always start with capital letters